### DISCUSSION TITLE
ZLUDA Missing module

### DISCUSSION THREAD

**Discussion by fraxyl** [vladmandic/sdnext#2873] [Q&A] [answered]
Following the ZLUDA install guide from the wiki and got to the last part running webui and I get this error:

E:\automatic\venv\lib\site-packages\torch\__init__.py:133 in <module>                                               │
│                                                                                                                     │
│    132                 err.strerror += f' Error loading "{dll}" or one of its dependencies.'                        │
│ >  133                 raise err                                                                                    │
│    134                                                                                                              │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
OSError: [WinError 126] The specified module could not be found. Error loading "E:\automatic\venv\lib\site-packages\torch\lib\cublas64_11.dll" or one of its dependencies.

![image](https://github.com/vladmandic/automatic/assets/159160137/32304ec9-bdf9-4c3e-a92d-3c1514609bca)

**Comment by fraxyl** [comment]
I forgot to add the HIP to my path. 🤦‍♂️

**Comment by fraxyl** [comment]
After adding %HIP_PATH%bin to my path, it still gives the same error, so I can rule that out.

**Reply by iDeNoh** [reply, marked as answer]
if you haven't already, please hop on to the discord and check out the Zluda chat under #help. https://discord.gg/c3k5WszV
